fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -506,13 +511,44 @@ public Response createObject(@HeaderParam("Content-Disposition") final ContentDi
             session.save();
         } catch (final RepositoryException e) {
             throw new RepositoryRuntimeException(e);
+        } catch (final Exception e) {
```

Comment:
In this case,  there is a problem I now see.  BinaryStoreException is actually a subclass of RepositoryException, so I should really remove lines 512 and 513.